CMakeLists.txt

```diff
@@ -47,13 +47,6 @@ include(cmake/gaudi_binary_tag.cmake)
 
 find_package(GaudiProject)
 
-configure_file(cmake/gmprun.in ${CMAKE_BINARY_DIR}/gmprun)
```

Comment:
Then you can also remove the file `cmake/gmprun.in`, right?

Reply:
True, good catch